examples/plot_bcic_iv_2a_moabb_cropped.py

```diff
@@ -0,0 +1,135 @@
+"""
+Cropped Decoding on BCIC IV 2a Competition Set with skorch and moabb.
+=====================================================================
+"""
+
+# Authors: Maciej Sliwowski <maciek.sliwowski@gmail.com>
+#          Robin Tibor Schirrmeister <robintibor@gmail.com>
+#          Lukas Gemein <l.gemein@gmail.com>
+#          Hubert Banville <hubert.jbanville@gmail.com>
+#
+# License: BSD-3
+from collections import OrderedDict
+from functools import partial
+
+import numpy as np
+import torch
+import mne
+from skorch.callbacks import LRScheduler
+mne.set_log_level('ERROR')
+
+from braindecode.datautil.windowers import create_windows_from_events
+from braindecode.classifier import EEGClassifier
+from braindecode.datasets import MOABBDataset
+from braindecode.losses import CroppedNLLLoss
+from braindecode.models.deep4 import Deep4Net
+from braindecode.models.shallow_fbcsp import ShallowFBCSPNet
+from braindecode.models.util import to_dense_prediction_model, get_output_shape
+from braindecode.util import set_random_seeds
+from braindecode.datautil.signalproc import exponential_running_standardize
+from braindecode.datautil.transforms import transform_concat_ds
```

Comment:
```suggestion
from braindecode.datautil.transforms import concat_datasets
```

red flag is the fact that transform is repeated. The module name is part of the naming of the function

Reply:
hmhm, but here I think this suggestion would be highly confusing, like when you would call this function, e.g.:
`concat_datasets([list of functions to be applied])` ... 
it would be superconfusing to me (as it does not concat datasets but applies transforms to each dataset in a concatenated dataset).